hawkular-alerts-actions-plugins/hawkular-alerts-actions-email/src/main/webapp/WEB-INF/jboss-deployment-structure.xml

```diff
@@ -20,7 +20,7 @@
 <jboss-deployment-structure>
   <deployment>
     <dependencies>
-      <module name="org.hawkular.bus" />
+      <module name="deployment.hawkular-alerts-rest.war" />
```

Comment:
OK, will merge now, and will proceed with rebase of m+a PR, and will make that work again. We'll need to eventually solve the issue somehow such that Hm+a and standalone Ha both work.